selfdrive/locationd/locationd.cc

```diff
@@ -451,6 +544,8 @@ void Localizer::handle_msg(const cereal::Event::Reader& log) {
     this->handle_cam_odo(t, log.getCameraOdometry());
   } else if (log.isLiveCalibration()) {
     this->handle_live_calib(t, log.getLiveCalibration());
+  } else if (log.isGnssMeasurements()) {
```

Comment:
Lets disable processing of `gpsLocationExternal` when the first gnssMeasurement is seen.

Reply:
I want this pr to have the same behaviour as currently in master. Disabling gpslocationExternal would completely stop the working of location in the kalman filter since the messages from laika aren't working when not having internet.

Reply:
nvm. We want to disable gpslocationExternal such that we can confirm gnssmeasurements are working without it.